EDDI/App.xaml.cs

```diff
@@ -103,12 +103,9 @@ public static void StartRollbar()
         {
             // Configure Rollbar error reporting
 
-            // Generate or retrieve an id unique to this configuration for bug tracking
-            if (string.IsNullOrEmpty(Eddi.Properties.Settings.Default.uniqueID))
-            {
-                Eddi.Properties.Settings.Default.uniqueID = Guid.NewGuid().ToString();
-            }
-            _Rollbar.configureRollbar(Eddi.Properties.Settings.Default.uniqueID);
+            // Generate an id unique to this app run for bug tracking
+            if (!string.IsNullOrEmpty(Eddi.Properties.Settings.Default.uniqueID)) { Eddi.Properties.Settings.Default.uniqueID = null; }
```

Comment:
1. Is this desirable?
2. If so, does this mean we can do away with `Eddi.Properties.Settings.Default.uniqueID`?

Reply:
1. I believe so.
2. Yes.